nasl/nasl_scanner_glue.c

```diff
@@ -1005,6 +1008,118 @@ security_something (lex_ctxt *lexic, proto_post_something_t proto_post_func,
   return FAKE_CELL;
 }
 
+tree_cell *
+security_notus (lex_ctxt *lexic)
+{
+  tree_cell *result;
+  named_nasl_var *oid_var, *var, *vul_packages, *name, *version, *fixed,
+    *fixed_version, *specifier, *start, *end;
+  anon_nasl_var *pkg;
+  unsigned char *oid;
+  int i;
+  GString *result_buf, *buf;
+  gchar *kb_result;
+
+  result = get_variable_by_name (lexic, "result");
+  if (result == NULL)
+    {
+      nasl_perror (lexic, "security_lsc: oid or result is NULL\n");
+      return FAKE_CELL;
+    }
+
+  var = (named_nasl_var *) result->x.ref_val;
+
+  oid_var = get_var_by_name (&var->u.v.v_arr, "oid");
+  if (oid_var == NULL)
+    {
+      nasl_perror (lexic, "security_lsc: oid not found\n");
+      return FAKE_CELL;
+    }
+
+  oid = oid_var->u.v.v_str.s_val;
+
+  vul_packages = get_var_by_name (&var->u.v.v_arr, "vulnerable_packages");
+  if (vul_packages == NULL || vul_packages->u.var_type != VAR2_ARRAY)
+    {
+      nasl_perror (lexic, "security_lsc: vul_packages is not an array\n");
+      return FAKE_CELL;
+    }
+  result_buf = g_string_new (NULL);
+  for (i = 0; i < vul_packages->u.v.v_arr.max_idx; i++)
+    {
+      pkg = nasl_get_var_by_num (lexic, &vul_packages->u.v.v_arr, i, 0);
+      if (pkg == NULL)
+        continue;
+
+      name = get_var_by_name (&pkg->v.v_arr, "name");
+      version = get_var_by_name (&pkg->v.v_arr, "installed");
+      fixed = get_var_by_name (&pkg->v.v_arr, "fixed");
+      if (fixed == NULL)
+        continue;
+      fixed_version = get_var_by_name (&fixed->u.v.v_arr, "version");
+      specifier = get_var_by_name (&fixed->u.v.v_arr, "specifier");
+      start = get_var_by_name (&fixed->u.v.v_arr, "start");
+      end = get_var_by_name (&fixed->u.v.v_arr, "end");
+
+      if (name == NULL || version == NULL)
+        {
+          continue;
+        }
+
+      if (fixed_version != NULL && specifier != NULL)
+        {
+          int spec_len = 8 - (int) strlen ((char *) version->u.v.v_str.s_val);
+          buf = g_string_new (NULL);
+          g_string_printf (
+            buf,
+            "\nVulnerable package:%*s%s\n"
+            "Installed version:%*s%s-%s\n"
+            "Fixed version:%*s%s%s-%s\n",
```

Comment:
I think previously all packages names were aligned, now the fixed version is more right (not an issue for me, just a nitpicky fyi)
```
Vulnerable package:   bzip2-libs
Installed version:    bzip2-libs-1.0.8-8.el9.x86_64
Fixed version:          >=bzip2-libs-1.0.8-10.el9_5.x86_64
```

Reply:
Thanks, is fixed now :)